Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectBuilder/MSBuildProjectReference.cs

```diff
@@ -27,11 +34,7 @@ public sealed partial class MSBuildProjectReference : ScriptableObject
 
         [SerializeField]
         [Tooltip("Named profiles to configure different build options.")]
-        private MSBuildBuildProfile[] profiles = new[]
-        {
-            MSBuildBuildProfile.Create(name: "Build", autoBuild: true, arguments: "-t:Build -p:Configuration=Release"),
-            MSBuildBuildProfile.Create(name: "Rebuild", autoBuild: false, arguments: "-t:Rebuild -p:Configuration=Release"),
-        };
+        private MSBuildBuildProfile[] profiles;
```

Comment:
I assume if they are creating it manually, thy know what they are doing and will set them

I removed it so that failure of creating a csproj doesn't result in default build behaviour, as for many cases we don't want that behaviour.

Reply:
> I assume if they are creating it manually, they know what they are doing and will set them

I've been thinking of it more like let's optimize for the common case. If someone manually creates an MSBuildProjectReference, it is probably to point to a csproj outside of the Unity project. We should make this common case easy (by providing default profiles that are usually the ones you want).

> I removed it so that failure of creating a csproj doesn't result in default build behaviour, as for many cases we don't want that behaviour.

Can you explain this a bit more? I'm not understanding the scenario.

Reply:
Ok, I undid it.

The issue I was encountering is that if some error occurs during the import process, the entire import is seen as a failure and the default asset gets loaded which has the default build/rebuild/clean, etc.

Reply:
This should be fixed, GitHub is acting strange.